packages/widget-message-meet/README-USAGE.md

```diff
@@ -1,28 +1,109 @@
-# How to use the Message Meet Widget
+# Spark Message and Meet Widget
 
-Our widget is built using React (https://github.com/facebook/react), Redux (https://github.com/reactjs/redux), and the Spark JS SDK (https://github.com/ciscospark/spark-js-sdk).
+The Spark Message and Meet widget allows developers to quickly and easily incorporate messaging through Cisco Spark into an application. This widget handles the heavy lifting of coordinating between your application and the Spark APIs, and provides a complete Spark UI and messaging experience without having to build all front end UI yourself.
 
-Depending on how comfortable you are with these frameworks, there are are a number of ways you can use the widget on your website and in your code.
+Our widget is built using React <https://github.com/facebook/react>, Redux <https://github.com/reactjs/redux>, and the Spark JS SDK <https://github.com/ciscospark/spark-js-sdk>. This allows a developer to extend the UI to support different messaging contexts.
```

Comment:
maybe replace ". This allows" with ", allowing" and "to extend the" with "to extend their"